package.json

```diff
@@ -45,6 +45,7 @@
   },
   "homepage": "https://github.com/JoinColony/colonyNetwork#readme",
   "devDependencies": {
+    "@colony/colony-js-contract-loader-fs": "1.5.4",
```

Comment:
Does this even need to be in the root `package.json`? I suspect it is only used in the reputation miner

Reply:
It's used in `test/colony-network-mining.js`, `gasCosts` and will be in `test/colony-funding.js` after #294 is merged.